tasks/sass.js

```diff
@@ -36,6 +36,8 @@ module.exports = grunt => {
 					grunt.file.write(filePath, result.map);
 				}
 			}));
-		})().catch(grunt.fatal).then(done);
+		})().catch(error => {
+			grunt.fatal(error.formatted);
```

Comment:
The error here is not guaranteed to only be the Sass error, so the `.formatted` property may or may not exist.

Reply:
If I change to `error.formatted || error` ?

Reply:
Or, maybe, wrap `render` with `try ... catch` and throw error with formatted message?

Reply:
> If I change to error.formatted || error ?

👍